scipy/interpolate/_cubic.py

```diff
@@ -164,15 +164,16 @@ class PchipInterpolator(CubicHermiteSpline):
 
     Parameters
     ----------
-    x : ndarray
+    x : ndarray, shape (npoints, )
         A 1-D array of monotonically increasing real values. ``x`` cannot
         include duplicate values (otherwise f is overspecified)
-    y : ndarray
-        A 1-D array of real values. ``y``'s length along the interpolation
+    y : ndarray, shape (..., npoints, ...)
+        A 1-D array of real values. ``y``'s length along the *first*
```

Comment:
Does not have to be a 1D array. Also the length along the interpolation axis needs to match the length of `x`, not necessarily the first axis. Here and elsewhere.

Reply:
Changed 1-D to N-D and reverted to "interpolation"